sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -236,7 +236,9 @@ export { PollOperationState }
 export interface PurgeDeletedKeyOptions extends coreHttp.OperationOptions {
 }
 
-// @public
+// Warning: (ae-internal-missing-underscore) The name "RecoverDeletedKeyOptions" should be prefixed with an underscore because the declaration is marked as @internal
+// 
+// @internal
 export interface RecoverDeletedKeyOptions extends coreHttp.OperationOptions {
```

Comment:
`RecoverDeletedKeyOptions` is used only in an internal private method. Why are we exporting it?

Reply:
Moved to: https://github.com/Azure/azure-sdk-for-js/issues/5907#issuecomment-547576995